### DISCUSSION TITLE
Release schedule/roadmap?

### DISCUSSION THREAD

**Discussion by sonalita** [fluxcd/flux2#4444] [General] [answered]
Hi
We see that in Flux v2.1.2 our Aquasec tooling  is reporting a number of critical/high CVE issues relating to the docker and containerd layers.

I also see that there has been some dependabot commits to update dependencies.

My question is when is the next release date for Flux? I do not see anywhere a release schedule/roadmap.

I am being chased by our internal security to have these CVEs resolved as soon as possible  - hence my post here.

Thanks.

**Comment by stefanprodan** [comment, marked as answer]
- Release docs are here https://fluxcd.io/flux/releases/
- Release cadence and EOL here https://endoflife.date/flux/
- Next release planning issue here https://github.com/fluxcd/flux2/issues/4410

**Reply by stefanprodan** [reply]
I've add the link to release cadence to the roadmap in this PR https://github.com/fluxcd/website/pull/1726
